core/ecschema-editing/src/Editing/Editor.ts

```diff
@@ -72,6 +73,7 @@ export class SchemaContextEditor {
   public readonly unitSystems = new UnitSystems(this);
   public readonly propertyCategories = new PropertyCategories(this);
   public readonly invertedUnits = new InvertedUnits(this);
+  public readonly schemaItems = new SchemaItems(this);
```

Comment:
Why is schemaItems part of the public API?  It is confusing as we have editors for each individual schema item type.  The public API of the SchemaContextEditor is supposed to be user friendly and make it easy for users to do the right thing.